doc/BENCHMARKING.md

```diff
@@ -38,6 +38,10 @@ You can use the following steps.
         export S3_BUCKET_TEST_PREFIX=prefix_path/
         export S3_BUCKET_BENCH_FILE=bench_file_name
         export S3_BUCKET_SMALL_BENCH_FILE=small_bench_file_name
+        # to filter by job name; e.g. to only run small jobs
+        export S3_JOB_NAME_FILTER=small
+        # to skip setting up local EC2 storage for caching benchmarks; the value is not important
+        export S3_SKIP_STORAGE_SETUP=
```

Comment:
I wonder if we should skip it by default just to make manual run a bit easier.

Reply:
I'm optimizing for what runs more often which is the CI.

Reply:
But for the CI we only need 1 change to configure it correctly while we will have to configure it every time on manual runs.

Reply:
Alright. I'll change it.